charts/stable/gollum/templates/configmap-git-sync.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: git-sync
+  namespace: {{ .Release.Namespaces }} 
+data: 
+  git-sync: |
+{{ .Values.gitSync | indent 4 }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gollum-config
+  namespace: {{ .Release.Namespaces }} 
```

Comment:
Defining release namespace enforces something that is already set via helm. Not defining it allows postprocessors to make changes as needed.
```suggestion
```